packages/workbox-core/models/messages/messages.mjs

```diff
@@ -32,6 +32,22 @@ export default {
       `'${moduleName}.${className}.${funcName}()' must be an array.`;
   },
 
+  'not-of-type': ({paramName, expectedType, moduleName, funcName}) => {
```

Comment:
In the above code sample I included the ClassName. I know it's verbose, but would likely help diagnose issues (both for us and developers).

What do you think?

Reply:
Could have function name as 'constructor' or special case special case 'constructor' to nothing.

Reply:
Done.